install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -572,10 +572,15 @@ data:
   # https://github.com/cilium/hubble/blob/master/Documentation/metrics.md
   hubble-metrics: {{- range .Values.global.hubble.metrics.enabled }}
     {{.}}
-  {{- end }}
+{{- end }}
 {{- end }}
   # An additional address for Hubble server to listen to (e.g. ":4244").
   hubble-listen-address: {{ .Values.global.hubble.listenAddress | quote }}
+{{- if .Values.global.hubble.allowInsecure }}
```

Comment:
In addition, we also want to set  `hubble-allow-insecure: true` if `config.upgradeCompatibility==1.8`. See https://github.com/cilium/cilium/blob/88688c3410f9be0c4c64bf5317d259dff2e95541/install/kubernetes/cilium/charts/config/templates/configmap.yaml#L16

Reply:
See comment above.